go.mod

```diff
@@ -7,12 +7,12 @@ replace github.com/fluxcd/image-automation-controller/api => ./api
 require (
 	github.com/fluxcd/image-automation-controller/api v0.0.0-00010101000000-000000000000
 	github.com/fluxcd/image-reflector-controller v0.0.0-20200810165546-c2265d9b49b9
-	github.com/fluxcd/pkg/apis/meta v0.4.0
+	github.com/fluxcd/pkg/apis/meta v0.5.0
 	github.com/fluxcd/pkg/gittestserver v0.0.2
-	github.com/fluxcd/pkg/runtime v0.3.1
+	github.com/fluxcd/pkg/runtime v0.4.0
 	github.com/fluxcd/source-controller v0.3.0
 	// If you bump this, change TOOLKIT_VERSION in the Makefile to match
-	github.com/fluxcd/source-controller/api v0.3.0
+	github.com/fluxcd/source-controller/api v0.5.0
```

Comment:
This can be `v0.5.1`, saves us another line in the `go.sum` of `fluxcd/flux`.

Reply:
It imports both, but I don't want to upgrade source-controller/pkg past the libgit2 change yet.